core/test/unit/cdk-nag/nag-central-governance.test.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+/**
+ * Tests CentralGovernance
+ *
+ * @group unit/best-practice/central-governance
+ */
+
+import { Annotations, Match } from 'aws-cdk-lib/assertions';
+import { Role, CompositePrincipal, ServicePrincipal } from 'aws-cdk-lib/aws-iam';
+import { App, Stack, Aspects } from 'aws-cdk-lib';
+// eslint-disable-next-line import/no-extraneous-dependencies,import/no-unresolved
+import { AwsSolutionsChecks, NagSuppressions } from 'cdk-nag';
+import { CentralGovernance } from '../../../src/data-mesh'
+
+
+const mockApp = new App();
+
+const centralGovStack = new Stack(mockApp, 'centralGov');
+
+const lfAdminRole = new Role(centralGovStack, 'myRole', {
+  assumedBy: new CompositePrincipal(
+    new ServicePrincipal('glue.amazonaws.com'),
+    new ServicePrincipal('lakeformation.amazonaws.com'),
+    new ServicePrincipal('states.amazonaws.com')
+  ),
+})
+
+new CentralGovernance(centralGovStack, 'myCentralGov', {
+  lfAdminRole
+})
+
+Aspects.of(centralGovStack).add(new AwsSolutionsChecks());
+
+NagSuppressions.addResourceSuppressionsByPath(
```

Comment:
I think the role needs to be split into multiple purpose built roles to avoid wide permissions. The role is getting inline policies from the constructs so it should be tested

Reply:
Will be split in another release